library/src/lib/date-picker/date-picker.component.ts

```diff
@@ -35,47 +35,62 @@ import { DateFormatParser } from '../calendar/format/date-parser';
     encapsulation: ViewEncapsulation.None
 })
 export class DatePickerComponent implements OnInit, OnDestroy, ControlValueAccessor {
+    /** @hidden The value of the input */
     inputFieldDate = null;
+    /** @hidden Whether the date input is invalid */
     isInvalidDateInput: boolean = false;
+    /** @hidden Whether the date picker is open */
     isOpen: boolean = false;
+    /** @hidden Subject the calendar subscribes to when the date value from the datePicker component changes. For internal use. */
     dateFromDatePicker: Subject<string> = new Subject();
 
+    /** The type of calendar, 'single' for single date selection or 'range' for a range of dates. */
     @Input()
     type: CalendarType = 'single';
 
+    /** Date picker input placeholder string */
     @Input()
     placeholder: string = 'mm/dd/yyyy';
 
+    /** Whether this is the compact input date picker */
     @Input()
     compact: boolean = false;
 
+    /** The currently selected CalendarDay model */
     @Input()
     selectedDay: CalendarDay = {
         date: null
     };
 
+    /** Fired when the selectedDay model changes */
```

Comment:
more precise description, missing period (like in calendar, so i won't retag all these)